packages/components/pie-button/src/button.scss

```diff
@@ -1,3 +1,60 @@
-@use '@justeat/fozzie/src/scss/fozzie' as f;
+@import '@justeat/pie-design-tokens/dist/jet.scss';
```

Comment:
I don't know if we want to use `@import` here... I suspect you can still use `@use` but worth checking. AFAIK `@import` is deprecated